src/pages/advertiser/screens/Advertiser/Advertiser.tsx

```diff
@@ -17,7 +17,6 @@ const Advertiser = () => {
     const { showModal } = useModalManager();
     const { advertiserId } = useParams<{ advertiserId: string }>();
     const { data: advertiserInfo } = api.advertiser.useGetInfo();
-    const [showOverlay, setShowOverlay] = useState(false);
```

Comment:
we don't really need to show the overlay based off this state, we should check using the value returned from the advertiser_info subscription. With the fix from BE, this has been implemented, and it properly updates from BE side.